fs/copy.ts

```diff
@@ -190,17 +190,21 @@ async function copyDir(
   src = toPathString(src);
   dest = toPathString(dest);
 
+  const promisesCopyAsyncFunctions: Promise<void>[] = [];
```

Comment:
```suggestion
  const promises = [];
```
I feel like this would've been sufficient.